src/libs/domify.js

```diff
@@ -0,0 +1,9 @@
+// Get DOM node from HTML
+export default html => {
+	// Shortcut for html`text` instead of html(`text`)
+	if (html.raw) {
+		html = String.raw(...arguments);
+	}
```

Comment:
I don't like using tagged templates just for the sake of it. In this case it gives no added benefit over a normal function call, right?